as2/server/errors.md

```diff
@@ -0,0 +1,15 @@
+# Errors
+##### Messages pertaining to the error messages.
+---
+### Error
+Sent to a penguin to show a specific error. `e`
+
+|Argument|Type|Variadic?|
+|---|---|---|
+|Error ID|Unsigned integer|No|
+
+##### Error Types
+There is a large amount of error IDs in Club Penguin, here is a list of most(if not all) of them. https://pastebin.com/uGZ2QPAs
```

Comment:
The pastebin should be removed. The CP Protocol project should be self-contained and entirely accessible offline - we can document "official" error IDs (that the live CP server used) in a section at the end of the `errors` document.